tests/lib/acutest/acutest.h

```diff
@@ -820,6 +820,8 @@ test_dump_(const char* title, const void* addr, size_t size)
 void
 test_abort_(void)
 {
+    fflush(stdout);
```

Comment:
@PettitWesley I do not actually intend to merge this change as part of this PR. This is just so I can see the test output if it fails. I have a PR open against acutest to fix it for real. https://github.com/mity/acutest/pull/63

Reply:
So we might need to delete this part before merging it?

Reply:
Yes, my plan was to revert this once any review comments are addressed and everything else is good to go.

Reply:
I decided to just revert this now. I'll try to manage it locally.